packages/nestjs-module/src/interceptor/ogma.interceptor.ts

```diff
@@ -86,10 +90,21 @@ export class OgmaInterceptor implements NestInterceptor {
     }
   }
 
-  public log(logObject: string | LogObject, context: ExecutionContext): void {
+  public log(
+    logObject: string | LogObject,
+    context: ExecutionContext,
+    requestId?: string,
+  ): void {
     this.service.info(
       logObject,
       `${context.getClass().name}#${context.getHandler().name}`,
+      requestId,
     );
   }
+
+  public generateRequestId(): string {
```

Comment:
Was this made as a class method so that it could be overridden in other implementations? If so, the `OgmaInterceptor` should be removed from the `OgmaModule` when it comes to being bound via `APP_INTERCEPTOR` otherwise two interceptors for logging will be created. This will mean the user then needs to bind the `OgmaInterceptor` on their own, which would be a breaking change from the current functionality (though one I would be okay with).

 The other option is we allow a function to be passed as an option instead, then call `this.options.generateRequestId()`, but I like this class based approach.